saml_idp.gemspec

```diff
@@ -46,6 +46,7 @@ section of the README.
   s.add_dependency('activesupport', '>= 3.2')
   s.add_dependency('builder', '>= 3.0')
   s.add_dependency('nokogiri', '>= 1.6.2')
+  s.add_dependency('rexml')
```

Comment:
Do we need to add "rexml" for Ruby 3.0. Because, I have still could see "rexml" in ruby 3.0 doc. Please correct me if I missing anything.
https://docs.ruby-lang.org/ja/latest/library/rexml=2fdocument.html

Reply:
thanks.

please check this pull request. 
https://github.com/saml-idp/saml_idp/pull/155

rexml are not default gem (change to bundled gem) on ruby 2.8 (3.0) 
https://bugs.ruby-lang.org/issues/16485